datajoint/autopopulate.py

```diff
@@ -176,6 +177,8 @@ def populate(
         :param max_calls: if not None, populate at most this many keys
         :param display_progress: if True, report progress_bar
         :param processes: number of processes to use. Set to None to use all cores
+        :param return_success_count: if True, return the count of successful `make()` calls.
```

Comment:
the docstring appears to be missing a description of the return argument.

Reply:
Should we omit this input and always include the success count as part of the return dict?

Reply:
I'm a bit unclear what you mean here. The docstring is included there. 
Do you mean it is not descriptive enough?

Reply:
Oh, I see what you mean

Reply:
The docstring should include ` :return: ...`